parsing/location.ml

```diff
@@ -210,8 +211,15 @@ let print_loc ppf loc =
      existing setup of editors that parse locations in error messages (e.g.
      Emacs). *)
   comma ();
-  Format.fprintf ppf "%s %i" (capitalize "line")
-    (if line_valid line then line else 1);
+  Format.fprintf ppf "%s" (capitalize "line");
+  let startline_valid = if line_valid startline then startline else 1 in
+  let endline_valid = if line_valid endline then endline else startline_valid in
```

Comment:
`*_valid` are the names of predicate and booleans: "is the startline valid?". So the naming is wrong here. I would use `valid_{startline,endline}` instead, or `{startline,endline}_corrected`, or just shadow `{startline,endline}` locally if that works.